### PR TITLE
Add types for compression-next

### PR DESCRIPTION
**Pull request by AkiraMiyakoda**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/compression-next/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,174 @@
+// Type definitions for compression-next 1.0
+// Project: https://github.com/expressjs/compression
```
Comment (peterblazejewicz) [inline types/compression-next/index.d.ts:2]:
```ts
// Project: https://github.com/expressjs/compression
```
obviously, this is wrong and points to original package

File: types/compression-next/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+// Definitions by: Santi Albo <https://github.com/santialbo>
+//                 Rob van der Burgt <https://github.com/rburgt>
+//                 Neil Bryson Cargamento <https://github.com/neilbryson>
+//                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
+//                 Akira Miyakoda <https://github.com/AkiraMiyakoda>
```
Comment (peterblazejewicz) [inline types/compression-next/index.d.ts:7]:
```suggestion
// Definitions by: Akira Miyakoda <https://github.com/AkiraMiyakoda>
```
just add a reference to original DT package owners, but you'll be the owner.

**Comment by AkiraMiyakoda** [comment]
@peterblazejewicz Thanks for reviewing this PR.
I have fixed what you pointed out.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@AkiraMiyakoda thx!

**Comment by AkiraMiyakoda** [comment]
Ready to merge

**Pull request merged**